### PR TITLE
AI Functions

### PR DESCRIPTION
**Pull request by Rainb0wKey**

### CODE REVIEW

File: src/premium/aiQuota.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
+# $aiQuota
+Returns reaming token limit for the bot.
+
+## Syntax
+```
+$aiQuota
+```
+
+> 1 character = 1 token
```
Comment (kubastick) [inline src/premium/aiQuota.md]: Approximately 4 characters = 1 token

File: src/premium/aiQuota.md [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+
+## Example
+```
+$nomention
+Tokens reaming limit: $aiQuota
+```
+
+> Tokens resets every 24 hours!
```
Comment (kubastick) [inline src/premium/aiQuota.md]: Token usage is counted by adding all prompts over last 24 hours

**Review by kubastick — CHANGES_REQUESTED** [review]

**Comment by Rainb0wKey** [comment]
Done?

File: src/premium/aiQuota.md [hunk 3]
```diff
@@ -0,0 +1,34 @@
+# $aiQuota
+Returns reaming token limit for the bot.
```
Comment (samujjal-gogoi) [inline src/premium/aiQuota.md]: I suppose, here the correct word of `reaming` should be `remaining`.

Also, the same mistake is in the [Public API] as well.
@kubastick, please fix the typo there too.

[Public API]: https://botdesignerdiscord.com/public/api/function/$aiQuota

Comment (kubastick) [reply src/premium/aiQuota.md]: Done

**Review by kubastick — APPROVED** [review]

**Pull request merged**